dags/openshift_nightlies/config/install/rosa/ocm.json

```diff
@@ -0,0 +1,15 @@
+{
+    "aws_profile": "",
+    "aws_access_key_id": "",
+    "aws_secret_access_key": "",
+    "aws_authentication_method": "sts",
+    "aws_account_id": "",
+    "rosa_environment": "staging",
+    "rosa_cli_version": "master",
+    "ocm_environment": "stage",
+    "openshift_worker_count": 3,
+    "openshift_network_type": "OpenShiftSDN",
+    "openshift_worker_instance_type": "m5.2xlarge",
+    "machineset_metadata_label_prefix": "machine.openshift.io",
+    "openshift_workload_node_instance_type": "m5.2xlarge"
```

Comment:
If the dag dont use it, it should be ok to remove it, because it used ona script. Only vars used on the dag.py needs to be present on all the config files

Reply:
I removed openshift_workload_node_instance_type in latest commit and test is successful.So this is not needed for ocm testing.

http://venkataanil-ocm-testing-airflow.apps.sailplane.perf.lab.eng.rdu2.redhat.com/log?dag_id=4.10-rosa-ocm-api-load&task_id=api-load&execution_date=2022-06-03T11%3A23%3A08.710445%2B00%3A00